test/e2e/knative_serving_test.go

```diff
@@ -25,14 +25,15 @@ func TestKnativeServing(t *testing.T) {
 	})
 
 	t.Run("verify correct deployment shape", func(t *testing.T) {
-		// Check the desired scale of deployments in the knative serving namespace
-		for _, deployment := range []string{"activator", "controller", "autoscaler-hpa"} {
+		servingDeployments := []string{"activator", "autoscaler", "autoscaler-hpa", "controller", "domain-mapping", "domainmapping-webhook", "webhook"}
+
+		for _, deployment := range servingDeployments {
```

Comment:
When we refactor this loop, it is better to refactor the same codes in other places :sweat_drops: 
For example:
https://github.com/openshift-knative/serverless-operator/blob/ed2b76b759db261925e0fc6bd0d1788dc18b69e6/test/e2e/knative_serving_test.go#L51-L61
https://github.com/openshift-knative/serverless-operator/blob/ed2b76b759db261925e0fc6bd0d1788dc18b69e6/test/e2e/knative_eventing_test.go#L39-L49

Reply:
Not sure I want to touch other tests here. This goes beyond the description of this PR. Anyway I will fix it but should be part of another PR.